README.md

```diff
@@ -63,6 +64,13 @@ optional arguments:
                         Ignore tag, or more correct, a digest, if it is
                         referenced multiple times. ATTENTION: the default if
                         False!
+  -u BASICAUTHUSER, --basicauth-user BASICAUTHUSER
+                        The username, if the registry is protected with basic
+                        auth
+  -pw BASICAUTHPW, --basicauth-pw BASICAUTHPW
```

Comment:
Maybe we should make it possible to provide the user and password via environment variables and docker secrets if running in a container